otter/util/http.py

```diff
@@ -543,3 +545,16 @@ def try_json_with_keys(maybe_json_error, keys):
         return None
     else:
         return get_in(keys, error_body, None)
+
+
+def text(thing):
```

Comment:
Thanks for the details. While I completely agree with you I am not going to provide warning in this PR as there is no framework to provide that as of now. Its either info or err. I've created ticket #1959 to address this.